### DISCUSSION TITLE
the num of Instance quantity mismatch in Service Overview page and Instance page

### DISCUSSION THREAD

**Discussion by Wangyxiang** [apache/skywalking#10158] [Q&A]
In Service Overview page，the page shows 5 instance in charts.
But when I open the Instance page, it only shows 1 piece of data，Can tell me where the quantity is wrong?
follow is the chart in Service Overviews page
![image](https://user-images.githubusercontent.com/51821806/207764827-930e98be-174b-46c9-9587-cb25ca637ff5.png)
follow is the num of  instance page 
![image](https://user-images.githubusercontent.com/51821806/207765215-8bcbdfe7-d62c-416b-9f63-4f513cdf35cf.png)
the version of my used oap is v9.1 and the agent is 8.11
My 5 instances, on the agent side, are distinguished by using the same SW_AGENT_NAME and 5 different SW_AGENT_INSTANCE_NAME
![image](https://user-images.githubusercontent.com/51821806/207766250-f4d5c1dd-2ea1-401f-9dfc-346d5feffc3c.png)

**Comment by wu-sheng** [comment]
You could check the data on your own. This is not a question we could know what happens.

**Reply by Wangyxiang** [reply]
ok thanks
